### ISSUE TITLE
loaderUtils.parseQuery() is deprecated

### ISSUE THREAD

**Issue by eXaminator** [webpack-contrib/istanbul-instrumenter-loader#57]
We started using this loader a few days back and now receive a deprecation warning every time we run our tests. Here is the full waring plus the matching stacktrace:
```
(node:1873) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
    at Object.parseQuery (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-utils/index.js:78:3)
    at Object.module.exports (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/istanbul-instrumenter-loader/index.js:15:35)
    at LOADER_EXECUTION (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at iterateNormalLoaders (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:218:10)
    at /Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:233:3
    at context.callback (/Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/babel-loader/lib/index.js:159:14
    at /Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/babel-loader/lib/fs-cache.js:113:24
    at /Users/sweingaertner/Sites/standard/standard/vendor/ofcs/ofcs/node_modules/babel-loader/lib/fs-cache.js:44:14
    at Gunzip.onEnd (zlib.js:213:5)
    at emitNone (events.js:91:20)
    at Gunzip.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
```

**Comment by thetric** [comment]
This should be addressed in #51

**Comment by michael-ciniawsky** [comment]
Fixed by #61
